airflow/providers/dbt/cloud/CHANGELOG.rst

```diff
@@ -19,7 +19,15 @@
 Changelog
 ---------
 
-1.0.0
+1.0.2
+.....
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Fix mistakenly added install_requires for all providers (#22382)``
+
+1.0.1
```

Comment:
Why to remove 1.0.0?

Reply:
It was a mistake from the last changelog that I fixed now (we went straight to 1.0.1)